student.rb

```diff
@@ -2,12 +2,9 @@
 class Student < Person
   attr_accessor :classroom
 
-  def initialize(age, classroom:, parent_permission: true, name: 'Unknown')
-    super(age: age, parent_permission: parent_permission, name: name)
+  def initialize(age, name, classroom)
+    super(age, name)
     @classroom = classroom
-  end
-
-  def play_hooky
-    '¯\\(ツ)/¯'
+    classroom.add_student(self)
```

Comment:
- [ ] It would be nice to separate the method for this implementation from the initialize method and create a separate method that serves as the 'belongs-to' side (a student belongs to a classroom).
- The code below will help achieve that: 👍🏽
```ruby
def classroom=(classroom)
      @classroom = classroom
      classroom.students.push(self) unless classroom.students.include?(self)
  end
```